src/components/structures/HomePage.tsx

```diff
@@ -106,14 +108,18 @@ const HomePage: React.FC<IProps> = ({ justRegistered = false }) => {
     } else {
         const brandingConfig = config.branding;
         let logoUrl = "themes/element/img/logos/element-logo.svg";
+        let logoUrlSecondary = SettingsStore.getValue(UIFeature.FindSecondaryLogoImgUrl);
```

Comment:
Should FindSecondaryLogoImgUrl be called SecondaryLogoUrl and the local variable logoUrlSecondary be called secondaryLogoUrl?

Reply:
This one has been actioned.

Reply:
Please refer to pull request #11 for all new changes as git has closed this pull request automatically somehow and ability to redo pull request is disabled .